src/services/react-admin/patchDataProvider.js

```diff
@@ -0,0 +1,83 @@
gwO0OOkp067D4RhJrfMzGYIO0Rp+Z+c2z1Lncs3+KBpRUNbOiM5eu7I3d9L6xe74289d7IK/fGqnX3yS3zRu5ab6xc642yUjHOrU7Em6sT8EZeAl7Pi8frq+LxWm48Xs2NxyurEvHy2jS8XJqKl0pScaYkBaeLk3GqOBmnipJRV5iEE2uS8GxpPIo2BKFwfSDWrA/EmvJAFKwLRH6ZHvlleqxeq0deaQByS3XIKdVgeaka2SVqZBerkbNOg8JtWmzcr0flG84xEsUE4QmHavtZvif98Dyr7jT/5xsZxIPjLDBBmCAMJggdMEH4BxOECcJggtABE4R/MEGYIAwmCB0wQfgHE4QJwmCC0AEThH8wQZggDCYIHTBB+AcThAnCYILQAROEfzBBmCAMJghBajk8dcgV+3d2xvn8QcDMGGCGGWYOA1Li2+TH9Bh8YtDhn4/rcHFHEF45GYrjb7N9HEwQHrGpmsOpwh7413gf3AyV47ZOgXqVCuhg6lUq3A5U4cYwNT5eHoAXX2Z7PJggFLJnbyecye+JD5Ifw/VREtzVKjtcBmv5MVaDTww6/GFLEM68woRhgtiZjTUcdhxww4HtXfBWVl98M9yXmAxt1jBqFb6aqsWl7UF46UwoTrwVhmMXyQePLzBBrKWWw3O7PHBhfj98N1SKn4P8cU9NrpZoD3cCVLgVqcYn83R442gI8fDxASaIGTZXu6Dyya44u6I3PpoxGN8NlaJexS8hLNYsKhVuRqvx6bwA/HFzIF4+HYpjPL1HkAlia2o5bD3iij17O8FY3h1XZg/CL4H+xENsb2G+H63BB+sC8UpdCE6eY00xpxdkYw2HF9Z63a8hoqT42cmkMMfdABVuxqhxLVmHt58LxgtOfBSQUwmyc78bjq/1wkVDf3waL8LtAAXxMNJOvUqFX8LU+CJJhw/KAnG2NgTHnOikRccVpIbDjoNu2L+zM95c1gfXR0qIh81RuBWpxl9z9DhbG4JTZ0MduinmcILs3tsJ55Z64/ooCX4cIsMd7aPXEneUSnzs52eWnxStP+Ouhdd8KpXiv/4P15y7q1Tibw+8z12ldYMGd5RK/EMqbfHajpi0vKe9L8s3k7S4/GTQ/SFkCkLNBHmAqs1dcWXWIPwn2s8mQ69fyeXgOM4kLhyHt8XiVq/53t8f7mZe00gfNzeM6NYNW/r3xy0TkjXnW39/dHdxafH6l4RCq/7/b4pE6PTAZ//cxue164dEr8JX07T4cLUev6sMxnEH6LvwWpBN1S54Lae3zWetv5bL4eXq2gKPhrBaI4gLx8G9Ga4mZBG4u+NTmeyhBJns5dVmTXBXqUR4166tPs8WgrT6bJ0Kv6sM5nWfhb+C1HC4On2wzQu5MWRfyGQtmNerl9WCDOvWDVf9/Jr4H19fnBeJsKB3b3RuFvqgLl3MNpuaC+LS8PfuHIcLJj67OWd9fJre39XOgkB1f3Lyn4sCeCsJbwX5cNYguxSwOZb16WO1IAmenmbf55JEgkHu7k3BPTrYtPTNBZF6eKCfm1uTfLct9EWSevYEx3EI6dIFKg8PuwsC1f2RsM8ydTjBwxXHvBTkyJauNlkZS0IQqFTYOmBAU3Bn9eiBehOBby5IZNeuKO/XDxzHoauLC/4llZp836/kcng0q2l0nTsTEaSRKwV63o148U6QJw+74lawnKgcHS3Ia82aQeM8PXHPCkFuKhQQNtQ8M3r0aPX39UolZvfoAY7joPTwwE2Fgrggd7UqnDvCrzVgvBPk8JauxOXoaEGa9xNSe/Y0WTs+KMhdpRK53t5Nr/vAt+WK4s9lMng3NMOONTTbSAsClQrXZ2p51R/hnSB1xa1/LfkuyOZmTazNAwaY/BtTgnwmkzWNpi3p06fF369taIIFd+nS9G80CHIrUs2rvgjvBLmwoD9xOTpSkI/8/NDL1RUcx8Hbzc3sBKIpQaBSIb9vX3AcBy8XF3wlv9/0/Eoub/rbpwcOpEqQX0NUOHGeCWIz/mc22dGrjhDkF4UC12Qy1AkEUDQbWdr72GNmP8+cIJ/JZPBsECy5Z0/cUyqxuX9/cByHfm5u+K6ZcDQIckevQt3vmCA2489J5kNEqyBdXFwgdHeH0N0dg9zdm+YxGvFwccGuZr/0DyNIvUqFkd27N9VAX8vl0DSI8MwDwlEhSCATxKZ8kGI5SDQKYolpXl742M+vzc8zJwhUKlyTyZomHCMaZs0fc3fH1/KWo300CHI7SIU61sSyHe+l8U8Qgbs70nr2bGKCp2dTLeLbqRP+bmYew1pBoFJhopdXC/Gy+/RpNRrGBHECQf6YYXqUh2ZBTHXSlza8nuM4aDp3tjgbbo0grzYbKu7t6mry/WgQhHXSbczluXSMYmU9oiA3FAoMabaIMKxrV/xgYQl8W4LcUSqb+iI53t4m34MJ4gSCXDTQIUjjGidXjsPv2znMe0uhgH+zUaxFvXvjjhWLFU0JApUK38jl+JNE0mLkijZBfglV82rfCO8EeZeCeZC7SiWGdevWNAL1nqT1bkVr50E+9vNrsYy9oG9fq9ZiWbtZijpBwpggNuWdx/sSF+RLuRwDGpZx9HZ1xTUT+zgeZib9JaGwKbjdXFxQY2JFr6MI8vMQJohNeWux/QRZ178/tg8ciJd9fHBRIsE7YjH2DRrUJAfHccjs1cvk4sKHEaRepcK2AQOahmpdOA5vikSOKUg4E8Sm/O4J+wky1tPT4hxG83VOjyJII4aGTVgcx6GHqyv+r9k8hqMIciuCrcWyKeeW9rFbYc54YG6hEW83N2zo3x//lptfdt8eQW4qFJjS7DMVHh640dDhdhhB2GJF2/JGtv0EqW8I5mWJBGeEQpwSCvEniQT3CATLUfhpqJpXd5nwTpDXV/QmXsiMRxAkigliU15dyQThMzejmSA25eVVvYgXMuMRBIlR4/g7TBCb8eLqnsQLmdF+bgxjgtiU0wVMED5zYzi/BPl/9hKfH3RE1GkAAAAASUVORK5CYII=';
```

Comment:
Could you please use a lighter/smaller encoded picture to avoid add such a weight to commit history?


This _(`98 o`)_ blue square for example:

![index-fs8](https://user-images.githubusercontent.com/1846633/66065672-40423e80-e548-11e9-9578-443686259b55.png)


```suggestion
const defaultImage = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJ4AAACaAQMAAABi2mMuAAAAA1BMVEVHbJjlM5lCAAAAGklEQVRIx+3BMQEAAADCoPVPbQ0PoAAAAF4MDKIAAfsS1FsAAAAASUVORK5CYII=';
```